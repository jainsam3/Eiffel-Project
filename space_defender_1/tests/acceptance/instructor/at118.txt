--Move down right, collision occurs on right part during transit.
play(3,5,5,1)
fire
move(A,1)
move(B,5)
