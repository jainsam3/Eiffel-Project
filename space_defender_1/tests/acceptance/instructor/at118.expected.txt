  Welcome to Space Defender Version 1.
->play(3,5,5,1)
  state:1.0, ok
      1  2  3  4  5
    A _  _  _  _  _
    B S  _  _  _  _
    C _  _  _  _  _
->fire
  state:2.0, ok
  The Starfighter fires a projectile at: [B,1]
      1  2  3  4  5
    A _  _  _  _  _
    B S  *  _  _  _
    C _  _  _  _  _
->move(A,1)
  state:3.0, ok
  A projectile moves: [B,2] -> [B,3]
  The Starfighter moves: [B,1] -> [A,1]
      1  2  3  4  5
    A S  _  _  _  _
    B _  _  *  _  _
    C _  _  _  _  _
->move(B,5)
  state:4.0, ok
  A projectile moves: [B,3] -> [B,4]
  The Starfighter moves and collides with a projectile: [A,1] -> [B,4]
      1  2  3  4  5
    A _  _  _  _  _
    B _  _  _  X  _
    C _  _  _  _  _
  The game is over. Better luck next time!
